CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * drop Debian 8 (Jessie);
   * add Fedora 31, 32, 33, 34, 35;
   * add Ubuntu 21.04 (Hirsute).
+- Publish to RWS instead of S3.
```

Comment:
Are you sure a user will understand this change with such description?

Reply:
Well, I'm not sure. Can it affect users at all?

Reply:
*I also think this should not be added to the CHANGELOG
* We continue to publish our packages on S3, but using a different tool

Reply:
Removed entry